README.md

```diff
@@ -99,10 +104,24 @@ sudo npm install -g ijavascript
 ijsinstall
 ```
 
-In Windows, [Anaconda](http://continuum.io/downloads) offers a convenient
-distribution to install Python and many other packages, such as Jupyter and
-IJavascript.
+### Windows
+Install [Python3](https://www.python.org/downloads/) or the [Anaconda3](http://continuum.io/downloads) Python distribution.
+
+In the command line:
+```
+pip3 install --upgrade pip
+pip3 install jupyter
+npm install -g ijavascript
+ijsinstall
+```
 
+If the `ijinstall` command is not recognized, you can execute it manually by navigating to your npm install directory and running the `ijinstall` batch file. If you do not know where your npm install is located, try looking for it in the default install location: `C:\Users\USERNAME\AppData\Roaming\npm`.
+
+Then you can run `jupyter notebook` in your terminal to load Jupyter Notebook. When you create a new Jupyter Notebook, you should see the Javascript (Node) kernel available.
+
+If using the [Anaconda](http://continuum.io/downloads) alternative to the standard [Python distribution](https://www.python.org/downloads/), it comes pre-installed with Jupyter Notebook. If using Anaconda, you can skip the `pip3 install jupyter` step.
+
```

Comment:
Please, remove *"If using Anaconda, you can skip the `pip3 install jupyter` step."* and write the instructions for Anaconda; e.g.:

Open the `Anaconda Prompt` and run:
```
conda install nodejs
npm install -g ijavascript
ijsintall
```